src/components/Reporting/PopupReport.tsx

```diff
@@ -0,0 +1,257 @@
+    height: "20px",
+    marginLeft: "12px",
+    ...theme.font16_bold,
+    lineHeight: "20px",
+    letterSpacing: "0.1em",
+    color: theme.black,
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+    whiteSpace: "nowrap",
+    width: "200px",
+  };
+  const styleTop: React.CSSProperties = {
+    marginTop: "6px",
+    width: "100%",
+    display: "flex",
+    alignItems: "center",
+  };
+  const styleMiddle: React.CSSProperties = {
+    margin: "18px 20px 0px",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    marginBottom: "6px",
+  };
+  const styleLabel: React.CSSProperties = {
+    width: "2.5em",
+    ...theme.font14,
+    lineHeight: "16px",
+    letterSpacing: "0.05em",
+    color: theme.gray_text,
+  };
+  const styleDropdown: React.CSSProperties = {
+    width: "100%",
+    marginLeft: "10px",
+    height: "28px",
+    background: theme.gray_background,
+    boxShadow: theme.shadow_gray_input_inset,
+    borderRadius: "6px",
+    outline: "none",
+    border: "none",
+    paddingLeft: "12px",
+    ...theme.font14,
+    lineHeight: "16px",
+    color: theme.gray_text,
+  };
+  const styleBottom: React.CSSProperties = {
+    marginTop: "10px",
+    display: "flex",
+    gap: "10px",
+    justifyContent: "space-between",
+  };
+  const styleBottomSubmitted: React.CSSProperties = {
+    margin: "32px",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center",
+    gap: "10px",
+  };
+  const styleCancel: React.CSSProperties = {
+    width: "77px",
+    height: "36px",
+    background: theme.gray_background,
+    boxShadow: theme.shadow_gray_button_inset,
+    borderRadius: "8px",
+    color: theme.gray_text,
+  };
+  const styleSubmit: React.CSSProperties = {
+    width: "218px",
+    height: "36px",
+    background: "#6E3678",
+    boxShadow: theme.shadow_gray_button_inset,
+    borderRadius: "8px",
+    color: "white",
+  };
+  const styleETC: React.CSSProperties = {
+    display: "flex",
+    justifyContent: "space-between",
+    background: theme.gray_background,
+    boxShadow: theme.shadow_gray_input_inset,
+    borderRadius: "6px",
+    margin: "10px 20px 0px",
+  };
+  const styleText: React.CSSProperties = {
+    background: theme.gray_background,
+    width: "221px",
+    minHeight: "16px",
+    lineHeight: "16px",
+    fontSize: "14px",
+    outline: "none",
+    border: "none",
+    resize: "none",
+    margin: "8px 8px 8px 0px",
+    overflow: "hidden",
+  };
+  const styleIcon: React.CSSProperties = {
+    position: "relative",
+    top: "10.75px",
+    left: "13.75px",
+    width: "10.5px",
+    height: "10.5px",
+  };
+
+  const handleType = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setType(event.target.value as Types);
+  };
+
+  const handleSubmit = async (): Promise<void> => {
+    const data: Data = {
+      reportedId: reportedId,
+      type: type,
+      etcDetail: etcDetail,
```

Comment:
attribute 이름이랑 변수명이랑 같을 땐 생각 가능합니다!
```suggestion
      reportedId,
      type,
      etcDetail,
```

Reply:
우와 attribute 이름을 일부만 생략해도 되는지 처음알았네요!